src/libraries/Native/Unix/System.Globalization.Native/pal_icushim.c

```diff
@@ -18,6 +18,8 @@
 
 #include "pal_icushim.h"
 
+#ifndef DISABLE_SHARED
```

Comment:
Why use this ifdef. Could we instead use a more descripting if def like, TARGETS_BROWSER?

Reply:
Yeah I'm not sure what this is used for either, @vargaz was this just for testing?

Reply:
I tried locally without it and it worked fine, removed it for now so we can get this PR in.

Reply:
How can `Load_ICU` work? It uses `dlopen` is that even supported in wasm?

Reply:
wasm supports dlopen, though we're not using it

Reply:
it's not, but we're running in Invariant mode still. That will be addressed by Egor/Katelyn's PR.

Reply:
Ok, cool